tests/integration/dynamic-controllers-express-server.spec.ts

```diff
@@ -600,6 +667,47 @@ describe('Express Server', () => {
+          expect(body.fields['body.nestedObject.intValue'].value).to.equal(bodyModel.intValue);
+          expect(body.fields['body.nestedObject.longValue'].message).to.equal('Custom Required long number.');
+          expect(body.fields['body.nestedObject.longValue'].value).to.equal(bodyModel.longValue);
+          expect(body.fields['body.nestedObject.booleanValue'].message).to.equal('invalid boolean value');
+          expect(body.fields['body.nestedObject.booleanValue'].value).to.equal(bodyModel.booleanValue);
+
+          expect(body.fields['body.nestedObject.dateValue'].message).to.equal('invalid ISO 8601 date format, i.e. YYYY-MM-DD');
+          expect(body.fields['body.nestedObject.dateValue'].value).to.equal(bodyModel.dateValue);
+          expect(body.fields['body.nestedObject.datetimeValue'].message).to.equal('invalid ISO 8601 datetime format, i.e. YYYY-MM-DDTHH:mm:ss');
+          expect(body.fields['body.nestedObject.datetimeValue'].value).to.equal(bodyModel.datetimeValue);
+
+          expect(body.fields['body.nestedObject.numberMax10'].message).to.equal('max 10');
+          expect(body.fields['body.nestedObject.numberMax10'].value).to.equal(bodyModel.numberMax10);
+          expect(body.fields['body.nestedObject.numberMin5'].message).to.equal('min 5');
+          expect(body.fields['body.nestedObject.numberMin5'].value).to.equal(bodyModel.numberMin5);
+          expect(body.fields['body.nestedObject.stringMax10Lenght'].message).to.equal('maxLength 10');
+          expect(body.fields['body.nestedObject.stringMax10Lenght'].value).to.equal(bodyModel.stringMax10Lenght);
+          expect(body.fields['body.nestedObject.stringMin5Lenght'].message).to.equal('minLength 5');
+          expect(body.fields['body.nestedObject.stringMin5Lenght'].value).to.equal(bodyModel.stringMin5Lenght);
+          expect(body.fields['body.nestedObject.stringPatternAZaz'].message).to.equal("Not match in '^[a-zA-Z]+$'");
+          expect(body.fields['body.nestedObject.stringPatternAZaz'].value).to.equal(bodyModel.stringPatternAZaz);
+
+          expect(body.fields['body.nestedObject.arrayMax5Item'].message).to.equal('maxItems 5');
+          expect(body.fields['body.nestedObject.arrayMax5Item'].value).to.deep.equal(bodyModel.arrayMax5Item);
+          expect(body.fields['body.nestedObject.arrayMin2Item'].message).to.equal('minItems 2');
+          expect(body.fields['body.nestedObject.arrayMin2Item'].value).to.deep.equal(bodyModel.arrayMin2Item);
+          expect(body.fields['body.nestedObject.arrayUniqueItem'].message).to.equal('required unique array');
+          expect(body.fields['body.nestedObject.arrayUniqueItem'].value).to.deep.equal(bodyModel.arrayUniqueItem);
+          expect(body.fields['body.nestedObject.model'].message).to.equal('invalid object');
+          expect(body.fields['body.nestedObject.model'].value).to.deep.equal(bodyModel.model);
+          expect(body.fields['body.nestedObject.mixedUnion'].message).to.equal(
```

Comment:
We should have a negative assertion for `intersection` here too.

Reply:
https://github.com/lukeautry/tsoa/pull/415/commits/d0fe540cc6734d10a10b189fe3c24c4722979505

Reply:
Thanks for your review. I added intersection and nested intersection specs, both valid and invalid for all the controllers that have these tests.